### PR TITLE
Updated crucial changes to README

### PR DESCRIPTION
**Pull request by SpencerMAQ**
@SpencerTest 
will you check if this is okay? this looks good to me

Are the placements of the lines okay or to fancy? 👍 
codecogseqn

# #60f8b11cfd50e5eb23d5415f89e07454d4ef6e77

- [ 11] 
- 

- [x] [3213]
- [x] [3213]
- [ ] [3213]
- [ ] [3213]
- [ ] Solve Navier Stokes

- [ ] P=NP

- [x] Solve Riemann Hypothesis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# OreillyTrainingTest
\ No newline at end of file
+# OreillyTrainingTest
+
+Just trying some stuff
+
+Maybe this line should go here
+
```
Comment (SpencerTest) [inline README.md]: You forgot to add on this line that you've already solved the Riemann Hypothesis.
😉

Comment (SpencerMAQ) [inline README.md]: Oh yeah god

**Comment by SpencerMAQ** [comment]
Hey Dude! @SpencerTest 

Thanks for spending 100days for imporving README, you deserve the Millenium Prize. There's just something I'd like you to consider.

File: README.md [hunk 2]
```diff
@@ -7,3 +7,5 @@ Maybe this line should go here
 or here
 
 or here
+
+A Winnar Is yoou
```
Comment (SpencerTest) [inline README.md]: Last thing, it's winRar not Winnar.
Thanks again!
:thailand: 
🍰

Comment (SpencerMAQ) [inline README.md]: Of course 📄

**Comment by SpencerTest** [comment]
👍 
Love you bro
MERGED

**Pull request merged**